README.md

```diff
@@ -1,26 +1,85 @@
 # @sanity/client
 
+[![npm stat](https://img.shields.io/npm/dm/@sanity/client.svg?style=flat-square)](https://npm-stat.com/charts.html?package=@sanity/client)
 [![npm version](https://img.shields.io/npm/v/@sanity/client.svg?style=flat-square)](https://www.npmjs.com/package/@sanity/client)
-
-Javascript client for Sanity. Works in node.js and modern browsers (older browsers need a [Promise polyfill](https://www.sanity.io/help/js-client-promise-polyfill)).
+[![gzip size][gzip-badge]][bundlephobia]
+[![size][size-badge]][bundlephobia]
+
+Javascript client for Sanity. Works in [Node.js](https://nodejs.org/en/), [Bun], [Deno], [Edge Runtime] and [modern browsers].
+
+# Table of contents<!-- omit in toc -->
+
+- [@sanity/client](#sanityclient)
+  - [Requirements](#requirements)
+  - [Installation](#installation)
+  - [API](#api)
+    - [Creating a client instance](#creating-a-client-instance)
```

Comment:
Table of contents is awesome, and having detailed runtime instructions is great! (but) I find that big ones like this can look intimidating to first time users. 

Could we perhaps extract some of the finer details into `docs/<feature-details>.md` files and link them instead?  Or, consider putting install instructions and esm example on top?
Ie: put the common happy path on top so not to overwhelm anyone?

Reply:
Added it in a new commit. Please review @stipsan and @snorrees!

Reply:
I like everything on one page and jumping between tabs is by far the most overwhelming thing I have to deal with every day. But I'm fine with being the outlier 🙇 
Looks good to me @kmelve!

Reply:
Just realized that there is a complexity with this document also being the source of https://www.sanity.io/docs/js-client 🤔